data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -347,6 +347,7 @@ then
 		${CLOUD_CREDENTIAL_OPERATOR_IMAGE} \
 		render \
 			--dest-dir=/assets/cco-bootstrap \
+			--mannifests-dir=/assets/manifests \
```

Comment:
`mannifests` -> `manifests`

Reply:
nice! where in the failed CI artifacts could i have seen that message?

Reply:
> https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/3381/pull-ci-openshift-installer-master-e2e-aws/10911/artifacts/e2e-aws/ipi-install-install/log-bundle-20200402143749.tar
```
bootstrap/journals/bootkube.service
```